### DISCUSSION TITLE
limit the maximum number of pages that can be saved in "Book_and_Quill"

### DISCUSSION THREAD

**Discussion by CatTeaA** [EssentialsX/Essentials#5612] [Ideas and Feature Suggestions] [answered]
Request a new feature to limit the maximum number of pages that can be saved in Book and Quill.
When the limit of pages is exceeded, it will not be saved, and you will be notified accordingly.
There are so many bad things that "Book and Quill" can do, and it's clear that limiting the number of pages you can save is beneficial

**Comment by JRoy** [comment, marked as answer]
This is really outside the scope of EssentialsX and probably better scoped for an Anti-Cheat or bugs to be fixed by the server.
